src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.image-link {
+  display: block;
+  height: 40px;
+  padding: 10px 0;
+  margin-left: 50px;
+}
+
+.nav {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin-right: 40px;
+}
+
+.menu {
+  display: flex;
+  list-style: none;
+  padding-inline-start: 0;
+  justify-content: flex-start;
+  margin: 0;
+}
+
+.menu-option {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding: 0 10px;
```

Comment:
Padding to link container will not increase link clickable zone, so I don't see any reason to use padding instead of margin but ok.
But setting padding for both sides of element is strange, especially when you have to calculate parent's margin to include this value.

Set padding/margin to one side and disable it for first/last element, don't make it overcomplicated